hardhat/README.md

```diff
@@ -0,0 +1,67 @@
+# Klaytn ServiceChain Token Bridge with Hardhat
+
+This project ports ServiceChain token test/deploy to hardhat.
+
+## Testing on Hardhat network
+Run the commands:
+
+```
+npm install
+npx hardhat run scripts/deploy_and_transfer.js
+```
+
+Note that it deploys both bridge contracts and ServiceChain tokens on the same network, so it only gives you an idea how the whole procedure works.
+
+## Testing on Klaytn test network
+### Prerequisites
+- Run `npm install`
+- Include `eth` to `RPC_API` in `kend.conf` and `kscnd.conf`. The feature requires klaytn >= v1.8.0. Also, Klaytn should be restarted after you modify the configuration files.
```

Comment:
```suggestion
- Include `eth` to `RPC_API` in `kend.conf` and `eth,subbridge` to `RPC_API` in `kscnd.conf`. The feature requires klaytn >= v1.8.0. Also, Klaytn should be restarted after you modify the configuration files.
```

To use `subbridge` API, we need to enable it first.

Reply:
done